packages/amazon/src/serverGroup/configure/serverGroupConfiguration.service.ts

```diff
@@ -376,14 +380,6 @@ export class AwsServerGroupConfigurationService {
     const result: IAmazonServerGroupCommandResult = { dirty: {} };
 
     if (command.region && (command.virtualizationType || command.viewState.disableImageSelection)) {
-<<<<<<< HEAD
-      let filtered = this.awsInstanceTypeService.getAvailableTypesForRegions(command.backingData.instanceTypes, [
-        command.region,
-      ]);
-      if (command.virtualizationType) {
-        filtered = this.awsInstanceTypeService.filterInstanceTypes(
-          filtered,
-=======
       let filteredTypesInfo: IAmazonInstanceType[] = this.awsInstanceTypeService.getAvailableTypesForRegions(
         command.backingData.instanceTypesInfo,
         [command.region],
```

Comment:
Still need to fix the merge conflicts in this file

Reply:
![sigh](https://i.gifer.com/4UiS.gif)

Reply:
![4UiS](https://user-images.githubusercontent.com/1945383/232884635-b37a0852-60fa-4914-8fdb-5b4524b14806.gif)